webpack/prod.js

```diff
@@ -22,6 +23,17 @@ module.exports = [
       filename: "[name].js",
       publicPath
     },
+    optimization: {
+      splitChunks: {
+        chunks: 'all',
+        cacheGroups: {
+          vendor: {
+            test: /[\\/]node_modules[\\/]/,
+            name: 'vendors'
+          }
+        }
+      }
+    },
```

Comment:
let's talk about `1.styles.css`: when looking at it's contents, it looks like it's only the blueprint styles... but those styles are also present in `styles.css`. What can we do about this duplication?